frontend/src/components/auth/Login.tsx

```diff
@@ -25,22 +25,31 @@ const Login = (): React.ReactElement => {
   const [email, setEmail] = useState("");
   const [password, setPassword] = useState("");
   const [loginError, setLoginError] = useState(false);
```

Comment:
can we rename `loginError` -> `invalidFormError`

Reply:
made the change

Reply:
oh sorry i meant the old one should be named `invalidFormError` (for the unfilled fields)
and the general error is `loginError`